widget/slider.go

```diff
@@ -0,0 +1,273 @@
+package widget
+
+import (
+	"image/color"
+	"math"
+
+	"fyne.io/fyne"
+	"fyne.io/fyne/canvas"
+	"fyne.io/fyne/theme"
+)
+
+const (
+	maxSliderDecimals = uint8(12)
+	standardScale     = 6
+	minLongSide       = 150
+)
+
+var (
+	_ fyne.Draggable = (*Slider)(nil)
+
+	activeTrackColor color.Color
```

Comment:
The move to global vars breaks the ApplyTheme() functionality I think.
Themes can change at runtime. If you load up in the fyne_demo app you can test this.

Reply:
strange that it doesn't show up when I run "go test" locally, but that is an easy enough revert.